platform/import-export/src/actions/ImportAsJSONAction.js

```diff
@@ -115,8 +116,7 @@ define(['zepto', 'objectUtils'], function ($, objectUtils) {
                 delete newModel.persisted;
 
                 newObj = this.instantiate(newModel, keystring);
-                newObj.getCapability("location")
-                    .setPrimaryLocation(tree[keystring].location);
+                this.openmct.objects.save(newModel);
```

Comment:
I think child `.location` needs to be set here? That's what the `locationCapability` used to do.

Reply:
I think its part of JSON

Reply:
so before start anything, `generateNewIdentifiers` does `rewriteId` which basically is `tree = JSON.stringify(tree).replace(new RegExp(oldIdKeyString, 'g'), newIdKeyString);`